app.py

```diff
@@ -120,6 +125,13 @@ def cli(ctx=None, verbose=False, pypi_rss_feed=None, graph_hosts=None, graph_por
         only_if_package_seen=only_if_package_seen
     )
 
+    push_gateway = os.getenv('PROMETHEUS_PUSH_GATEWAY')
```

Comment:
could we set a default value for this?

Reply:
I would default it to not pushing metrics if the env var is not set
for development purposes

Reply:
Do we need a default value (btw now it is None)?

Reply:
but depends on you, if you like we can default it to `http://pushgateway:9091` for the case that the pushgateway is deployed in a local NS

Reply:
> but depends on you, if you like we can default it to http://pushgateway:9091 for the case that the pushgateway is deployed in a local NS

Also note that the `app.py` script can be run as a CLI - I don't see any reason why we should have this default URL. I would suggest to report that metrics were not pushed in an else branch (using `_LOGGER.warning()`) if env variable is not set so we see that metrics were sent/not sent when there will be some issues.